_data/global_leadership.yml

```diff
@@ -0,0 +1,9 @@
+- name: Jay Miller
+  title: Executor
+  image: https://github.com/kjaymiller.png
+  alt: photo of Jay
+
+- name: Velda Kiara
+  title: Vice Executor
+  image: https://github.com/VeldaKiara.png
+  alt: photo of Velda
```

Comment:
```suggestion
```

Velda is actually stepping down and these roles are effectively taking the place of the Vice Executor Role.